src/Chess/Game/ChessGame.php

```diff
@@ -712,6 +712,9 @@ protected function getSquareFromParsedMove($parsedMove, $color = null)
                 }
                 $potentials = array();
                 foreach ($this->_pieces as $name => $value) {
+                    if (!$value) {
```

Comment:
Should we do `if ($value === false)` ? This library is not type strict at all :(

Reply:
Could be... for now I kept it consistent with the check in line 618